### DISCUSSION TITLE
Artemis 2.31.0

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#742] [Q&A] [answered]
I would like to know if possible when 2.31.0 artemis version will be supported by the operator.
I saw that with version 1.0.15 that latest and greatest version supported is 2.30.0.

**Comment by brusdev** [comment, marked as answer]
ArtemisCloud ActiveMQ Artemis Operator 1.0.16 thats support Apache ActiveMQ Artemis 2.31.2 has been released https://github.com/artemiscloud/activemq-artemis-operator/releases/tag/v1.0.16
